### PR TITLE
Regression: Fix handling of http requests in apps bridge

### PR DESCRIPTION
**Pull request by d-gubert**
## Proposed changes (including videos or screenshots)

Changes made during Meteor upgrade broke HTTP requests made in Rocket.Chat Apps

### CODE REVIEW

File: app/apps/server/bridges/http.ts [hunk 1]
```diff
@@ -78,16 +78,28 @@ export class AppHttpBridge extends HttpBridge {
...
-				}
+			const body = Buffer.from(await response.arrayBuffer());
+
+			if (request.encoding === null) {
+				/**
+				 * So, here we have a problem. The property is not appropriately typed
+				 * in the Apps-Engine. Thus, even though it was  previously supported to
+				 * return it as a Buffer. Apps ended up needing to use that same type
+				 * assertion you see below.
+				 */
```
Comment (d-gubert) [inline app/apps/server/bridges/http.ts:92]: @sampaiodiego would like your thoughts here on how to best handle the situation

Comment (d-gubert) [reply app/apps/server/bridges/http.ts:92]: Something interesting to add is that changing the type in the Apps-Engine can be a breaking change 😬

Comment (cauefcr) [reply app/apps/server/bridges/http.ts:92]: This fixes my inability to get a binary response from an http request in an app.

Comment (d-gubert) [reply app/apps/server/bridges/http.ts:92]: @cauefcr You should be able to do that in previous versions of Rocket.Chat 🤔

Comment (cauefcr) [reply app/apps/server/bridges/http.ts]: I was using a workspace on the develop branch like a dummy. :see_no_evil:

File: app/apps/server/bridges/http.ts [hunk 2]
```diff
@@ -78,16 +78,33 @@ export class AppHttpBridge extends HttpBridge {
...
 
-			if (request.hasOwnProperty('encoding')) {
-				if (request.encoding === null) {
-					result.content = Buffer.from(await response.arrayBuffer()).toString();
-				} else {
-					result.content = await response.text();
-				}
+			const body = Buffer.from(await response.arrayBuffer());
+
+			if (request.encoding === null) {
```
Comment (AllanPazRibeiro) [inline app/apps/server/bridges/http.ts:86]: maybe you can check if the encoding is not null by: 

`if (!request.encoding)`

**Review by AllanPazRibeiro — APPROVED** [review]

File: app/apps/server/bridges/http.ts (hunk 2 above)
Comment (d-gubert) [reply app/apps/server/bridges/http.ts:86]: Here we need a strict comparison, as `undefined` for example, is a valid parameter to the `Buffer.toString` function

**Review by sampaiodiego — APPROVED** [review]

**Pull request merged**